kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/OperatorMain.java

```diff
@@ -31,22 +36,23 @@
 public class OperatorMain {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(OperatorMain.class);
-    private MeterRegistry registry;
     private final Operator operator;
+    private final HttpServer managementServer;
 
-    public OperatorMain() {
-        this(null);
+    public OperatorMain() throws IOException {
+        this(null, HttpServer.create(new InetSocketAddress("0.0.0.0", 8080), 10));
```

Comment:
any motivation for the 10 backlog vs using the system default?

Reply:
That I needed to specify something so I could specify the socket address and I wanted to keep resource usage low.

Stepping through the call chain it looks like `backlog < 1` defaults to `50`. Can change this to `0` if preferred.

Reply:
yep 0 sounds good to me